synapse/config/emailconfig.py

```diff
@@ -108,9 +107,14 @@ def read_config(self, config, **kwargs):
             if self.trusted_third_party_id_servers:
                 # XXX: It's a little confusing that account_threepid_delegate_email is modified
                 # both in RegistrationConfig and here. We should factor this bit out
-                self.account_threepid_delegate_email = self.trusted_third_party_id_servers[
-                    0
-                ]  # type: Optional[str]
+
+                first_trusted_identity_server = self.trusted_third_party_id_servers[0]
+
+                # This config option does not contain a scheme whereas
```

Comment:
it's unclear what "this" refers to.

```suggestion
                # trusted_third_party_id_servers does not contain a scheme whereas
```